source/command-line-options.txt

```diff
@@ -33,18 +35,103 @@ Use the following syntax to start |compass| and set a configuration option:
 Options
 -------
 
-You can set the following configuration options using the command line:
+You can set these options when using the command line:
+
+.. option:: --autoUpdates
+  
+   Enable :ref:`automatic updates <enable-automatic-updates>`. Use 
+   ``--no-autoUpdates`` to disable automatic updates. 
+
+   Setting this option overrides the value in the :guilabel:`Settings` 
+   panel.
+  
+.. option:: --enableFeedbackPanel
+  
+   Enable the feedback panel. Use ``--no-enableFeedbackPanel`` to 
+   disable automatic updates. 
+
+   Setting this option overrides the value in the :guilabel:`Settings` 
+   panel.
+   
+.. option:: --enableMaps
+ 
+   Allow |compass| to make requests to a 3rd party mapping service. Use
+   ``--no-enableMaps`` to disable mapping requests.
+
+   Setting this option overrides the value in the :guilabel:`Settings` 
+   panel.
+
+.. option:: --exportConnections
+    
+   :ref:`Export <compass-export-connections-cli>` favorite connections.
+
+.. option:: --file <file>
+  
+   Specify a file containing a :ref:`list of connections 
+   <compass-export-connections-cli>`. These connections are 
+   automatically available once |compass| starts.
+
+.. option:: --help
+   
+   Show |compass| options.
+
+.. option:: --importConnections
+
+   :ref:`Import <compass-import-connections-cli>` favorite connections.
+
+.. option:: --lgDarkmode
+
+   Use leafygreen dark mode instead of darkreader.
+
+.. option:: --networkTraffic
+
+   Configure |compass| to only allow outgoing :ref:`network operations 
+   <compass-configure-network-traffic>` to connect to the database. 
+
+.. option:: --passphrase
+ 
+   Specify a passphrase to encrypt the :ref:`exported favorite 
+   connections <compass-export-connections-cli-encrypted>` output file.
+
+   Specify a passphrase to decrypt the :ref:`imported favorite
+   connections <compass-import-connections-cli-encrypted>` file. 
+
+.. option:: --protectConnectionStrings
+
+   Hide credentials in connection strings. Passwords in connection 
+   strings are displayed as `*****`. Toggle :guilabel:`Edit Connection 
+   String` to display the full, unprotected connection string.
```

Comment:
The point of this option is that you cannot do this anymore if it’s set :)

Reply:
Got it -- updated!